features/backend/products.feature

```diff
@@ -159,6 +156,18 @@ Feature: Products
         Then I should be on the page of product "Big Sticker"
         And I should see "Product has been successfully updated."
 
+    @javascript
+    Scenario: Modifying product attributes
+        Given I am editing product "Super T-Shirt"
+        When I go to "Attributes" tab
+        And I delete "T-Shirt fabric" attribute
+        And I add "Size" attribute
+        And I fill in "Size" with "100"
+        When I press "Save changes"
+        Then I should be on the page of product "Super T-Shirt"
+        And I should see "100"
```

Comment:
IMHO it is way too loose. It is very likely that page will contain "100" that isn't the size, but something else.